### PR TITLE
Updated the social links

### PR DESCRIPTION
**Pull request by J-CODE07**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5903"></a>

### CODE REVIEW

File: data/J-CODE07.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "tags": ["Open Source", "DevOps"],
   "Socials": [
     { "icon": "FaTwitter", "url": "https://twitter.com/jagan_1607" },
-    { "icon": "FaGithub", "url": "https://github.com/J-CODE07" }
+    { "icon": "FaGithub", "url": "https://github.com/J-CODE07" },
+    { "icon": "FaHashnode", "url": "https://j-code.hashnode.dev/" }
```
Comment (kumarsonsoff3) [inline data/J-CODE07.json:9]: You probably be using an invalid icon here. It should be `SiHashnode`.

```suggestion
    { "icon": "SiHashnode", "url": "https://j-code.hashnode.dev/" }
```

File: data/J-CODE07.json [hunk 2]
```diff
@@ -20,6 +21,13 @@
       "name": "GitHub - Open Source",
       "url": "https://github.com/J-CODE07",
       "icon": "FaGithub"
+    },
+
+    {
+      "group": "socials",
+      "name": "Hashnode - Blog",
+      "url": "https://j-code.hashnode.dev/",
+      "icon": "FaHashnode"
```
Comment (kumarsonsoff3) [inline data/J-CODE07.json:30]: Same here too. 

```suggestion
      "icon": "SiHashnode"
```

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**